kedro/templates/project/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -1,6 +1,5 @@
 black==21.5b1
 flake8>=3.7.9, <4.0
-ipython~=7.10
```

Comment:
This was just left over from a bad merge - the specification on the next line is the right one.